build/builder.sh

```diff
@@ -6,7 +6,7 @@ image="cockroachdb/builder"
 
 function init() {
   docker build --tag="${image}" - <<EOF
-FROM golang:1.4.2
+FROM golang:1.5
```

Comment:
Uh-oh, looks like the build-cache stuff isn't working properly. Could have sworn I put the go version number in there. You might need to rebuild with "clear-cache".